products/firefox_desktop/removing_locales.md

```diff
@@ -0,0 +1,53 @@
+# Removing a locale from Firefox for desktop
+
+Removing a locale from Firefox builds should be the last step in a long series of passages. A locale is removed when:
+* There’s no visible activity in Pontoon for a year.
+* The translation level of Firefox is dropping, causing a good portion of the user interface to appear in English.
```

Comment:
Ideally, this should have tiers percentage, but we don't have that information on official dashboards yet